docs/pages/build-reference/ios-capabilities.mdx

```diff
@@ -1,18 +1,20 @@
 ---
 title: iOS Capabilities
+description: Learn about built-in iOS capabilities supported in EAS Build and how to enable or disable them.
 ---
 
 import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
+import { Terminal } from '~/ui/components/Snippet';
 
 When you make a change to your iOS entitlements, this change needs to be updated remotely on Apple's servers before making a production build. EAS Build automatically synchronizes capabilities on the Apple Developer Console with your local entitlements configuration when you run `eas build`. Capabilities are web services provided by Apple, think of them like AWS or Firebase services.
 
 > This feature can be disabled with `EXPO_NO_CAPABILITY_SYNC=1 eas build`
 
 ## Entitlements
 
-In bare workflow apps, the entitlements are read from your `ios/**/*.entitlements` file.
+In bare workflow apps, the entitlements are read from your **ios/.entitlements** file.
```

Comment:
```suggestion
In bare workflow apps, the entitlements are read from your __ios/**/*.entitlements__ file.
```

I believe that glob for the file here was correct, but maybe this is a some kind of side-effect of switching to emphasized text via `**`. Alterative notation for bold text is `__` which would be better in this case.

Reply:
You are right about the side-effect. I didn't notice this. Great catch!